### PR TITLE
Improve title and image URL templating

### PR DESCRIPTION
**Pull request by ahuang11**
Just makes it easier.

### CODE REVIEW

File: {{cookiecutter.collection_name}}/README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-# {{ cookiecutter.collection_name }}
+# Coordinate and incorporate {{ cookiecutter.collection_name.split('-')[1:] | join | title}} in your dataflow with {{ cookiecutter.collection_name }}
```
Comment (zzstoatzz) [inline {{cookiecutter.collection_name}}/README.md:1]:
```suggestion
# Coordinate {{ cookiecutter.collection_name.split('-')[1:] | join | title}} in your dataflow with {{ cookiecutter.collection_name }}
```

"... incorporate gcp in your dataflow" for example, seems a bit weird

**Review by zzstoatzz — CHANGES_REQUESTED** [review]

File: {{cookiecutter.collection_name}}/README.md (hunk 1 above)
Comment (ahuang11) [reply {{cookiecutter.collection_name}}/README.md:1]: Coordinate GCP in your dataflow doesn't seem right either

Comment (ahuang11) [reply {{cookiecutter.collection_name}}/README.md:1]: What about "Integrate GCP with your dataflow using Prefect-GCP"?

Comment (zzstoatzz) [reply {{cookiecutter.collection_name}}/README.md]: i think "Coordinate GCP in your dataflow" makes logical sense, but I don't mind integrate either - could ask a product / marketing person which they prefer but its a horse a piece in my book

Comment (ahuang11) [reply {{cookiecutter.collection_name}}/README.md]: I like integrate

Comment (ahuang11) [reply {{cookiecutter.collection_name}}/README.md]: Tests are broken atm due to new Prefect release; need https://github.com/PrefectHQ/prefect-collection-template/pull/138/files too

**Pull request closed without merging**